libseccomp/src/lib.rs

```diff
@@ -1347,6 +1347,28 @@ pub fn set_api(level: u32) -> Result<()> {
     Ok(())
 }
 
+/// Resets libseccomp’s global state.
+///
+/// This function resets the (internal) global state of libseccomp, this
+/// includes any notification file descriptors retrieved by
+/// [`get_notify_fd`](ScmpFilterContext::get_notify_fd).
+/// Normally you do not need this but it may be required to continue using
+/// libseccomp after a `fork()`/`clone()` to ensure the API level and
+/// user notification state is properly reset.
```

Comment:
Nit: `libseccomp` -> `the libseccomp` in order to make the word consistent in the documentation.